akka-docs/src/main/paradox/stream/operators/Source-or-Flow/scanAsync.md

```diff
@@ -18,6 +18,15 @@ Just like `scan` but receives a function that results in a @scala[`Future`] @jav
 
 Note that the `zero` value must be immutable.
 
+## Example
+
+Scala
```

Comment:
I would want a short text explaining the sample "This sample will aggregate the values and emit the sum up to the current value for each value that comes in"  (even if that is perhaps obvious I think it makes for a nicer read)